internal/imap_filter/command/command.go

```diff
@@ -127,15 +128,23 @@ func (c *Check) expandCommand(msgMeta *module.MsgMetadata, accountName string) (
 				valI, err := msgMeta.Conn.RDNSName.Get()
 				if err != nil {
 					return ""
-				}
+				}x
 				if valI == nil {
-					return ""
+					return ""x
 				}
 				return valI.(string)
 			case "{msg_id}":
 				return msgMeta.ID
 			case "{sender}":
 				return msgMeta.OriginalFrom
+			case "{rcpts}":
+				rcpts := []string{}
+				for _, value := range msgMeta.OriginalRcpts {
+					rcpts = append(rcpts, value)
+				}
+				return strings.Join(rcpts, "\n")
+			case "{address}":
+				return msgMeta.OriginalRcpts[accountName]
```

Comment:
accountName is not guaranteed to be a email address at all.
E.g. https://maddy.email/multiple-domains/

IMAPFilter interface does not provide access to SMTP envelope information. I wonder how it should be (if should) extended to do.

Reply:
While I was testing I noticed that even when I send the email to multiple recipients rcpts would only have one single address. I do not not know if this is a bug in either my test or Maddy but it makes address to not being required.

On the other hand I also got one without  OriginalRcpts as the email was sent to my real email address (not a domain alias).

So it looks like having access to the original TO address is needed to make this more reliable...

Reply:
made the changes to pass the rctp_to